### PR TITLE
Milestone 2: User interface

### PR DESCRIPTION
**Pull request by Jhonatanssal**
### User interface
In this branch we added the code for the user interaction.
We use the prompts to get users' choices.

Happy coding!

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+puts '-------------'
+puts "| #{board[0]} | #{board[1]} | #{board[2]} |"
+puts '-------------'
+puts "| #{board[3]} | #{board[4]} | #{board[5]} |"
+puts '-------------'
+puts "| #{board[6]} | #{board[7]} | #{board[8]} |"
+puts '-------------'
+
+# Repeat user inputs until there is a winner
+
```
Comment (krys2fa) [inline bin/main.rb]: - [x] [REQUIRED] A loop is needed in this game (recommended is while but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png))
Please place a loop here which terminates after a winning or draw move. 
**NB:** Not everything has to work but the milestone should show that you have thought of every scenario in the game flow. The changes listed above are all that you are left with. 😃

**Review by krys2fa — CHANGES_REQUESTED** [review]
## Nice work @pbkabali and @jssarrazolaa  👏 
Your submission shows you have really thought through the game flow. 🥂 
Even though it is not required with this milestone, your project already has a good README. 👍 

The following are all that is left to complete the project. They can be seen below:
### Status: Changes Required ♻️ 

- [x] [REQUIRED] Your game should inform the players if the selected move is a winning move

- [x]  [REQUIRED] Your game should inform the players if the selected move is a draw move

- [x]  [REQUIRED] Your game should repeat all actions for the next player's move

**NB**: Kindly check the comment placed in your code below to see how to go about these changes.

Please request for another review after implementing the changes.

Happy Coding 🖥️  😃 

###### You can contact me on slack (Chris Amanor) for any clarification.

**Comment by pbkabali** [comment]
Hi @krys2fa, thank you very much for your review. We have worked on the feedback and submitted another review request

File: bin/main.rb [hunk 2]
```diff
@@ -14,14 +14,13 @@
 puts "| #{board[6]} | #{board[7]} | #{board[8]} |"
 puts '-------------'
 
-# Ask to Player 1 put X
-
+# until board.winner or board.draw
+# take input from player1
 puts 'Player 1, place X'
 player1 = gets.chomp.to_i
```
Comment (expjazz) [inline bin/main.rb]: - [optional] You should validate the user input (and display it the mistake)

**Review by expjazz — APPROVED** [review]
## Status: Project Approved :clap: 

You did a great job here :1st_place_medal: 

However, since it was not pointed out in the first review I'm making this as optional, so you can move on to the next milestone, but you should validate the user input.

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

File: bin/main.rb (hunk 2 above)
Comment (pbkabali) [reply bin/main.rb]: Oh yes, we understand. Thanks @expjazz

**Pull request merged**